src/main/java/net/licks92/WirelessRedstone/Strings/WirelessXMLStringsLoader.java

```diff
@@ -2,123 +2,234 @@
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.util.ArrayList;
 
 import javax.xml.parsers.DocumentBuilder;
 import javax.xml.parsers.DocumentBuilderFactory;
 import javax.xml.parsers.ParserConfigurationException;
 
+import net.licks92.WirelessRedstone.WirelessRedstone;
+
 import org.w3c.dom.Document;
 import org.w3c.dom.Element;
 import org.w3c.dom.NodeList;
 import org.xml.sax.SAXException;
 
-import net.licks92.WirelessRedstone.WirelessRedstone;
-
 /**
- * Loads the strings from a specific file. If some strings are missing, it will replace them by the one in the default english file.
+ * Loads the strings from a specific file. If some strings are missing, it will
+ * replace them by the one in the default english file.
  */
-public class WirelessXMLStringsLoader extends WirelessStrings
-{
+public class WirelessXMLStringsLoader extends WirelessStrings {
 	private final String STRINGS_FOLDER = "languages/";
 	private final String defaultLanguage = "en";
-	
-	public WirelessXMLStringsLoader(WirelessRedstone plugin, String language)
-	{
-		String defaultLocation = STRINGS_FOLDER + defaultLanguage + "/strings.xml";
-		InputStream stream = plugin.getResource(STRINGS_FOLDER + language + "/strings.xml");
-		if(stream != null) {
```

Comment:
So these are the 3 lines that were removed?
That's okay i will reimplement this and explain it in comments ;)